components/event-receiver/org.wso2.carbon.event.input.adapter.core/src/main/java/org/wso2/carbon/event/input/adapter/core/EventAdapterUtil.java

```diff
@@ -28,10 +32,12 @@ public static AxisConfiguration getAxisConfiguration() {
             axisConfiguration = InputEventAdapterServiceValueHolder.getConfigurationContextService().
                     getServerConfigContext().getAxisConfiguration();
         } else {
-            axisConfiguration = TenantAxisUtils.getTenantAxisConfiguration(CarbonContext.
-                            getThreadLocalCarbonContext().getTenantDomain(),
-                    InputEventAdapterServiceValueHolder.getConfigurationContextService().
-                            getServerConfigContext());
+            ConfigurationContext configurationContext = TenantConfigHolder.getTenantConfig(PrivilegedCarbonContext.getThreadLocalCarbonContext().getTenantId());
+            if(configurationContext != null){
+                axisConfiguration = configurationContext.getAxisConfiguration();
+            }else{
```

Comment:
Thanks @lasanthafdo, Done in pull request - https://github.com/wso2/carbon-analytics-common/pull/109